qa-tests-backend/src/test/groovy/ClusterInitBundleTest.groovy

```diff
@@ -50,7 +50,7 @@ class ClusterInitBundleTest extends BaseSpecification {
         assert response.initBundleRevokedIdsCount == 0
         and:
         "impacted cluster is listed"
-        assert response.initBundleRevocationErrorsList.first().impactedClustersList*.id.contains(clusterId)
+        assert response.initBundleRevocationErrorsList.first().impactedClustersList*.id.contains(cluster.id)
```

Comment:
I wonder how this ever worked the previous way 🙃

Reply:
`ClusterInitBundleTest` has 2 sub-tests. 
The CI logs show only the 2nd test `Test that cluster init bundle can be revoked when it has no impacted clusters` being executed.
The fixed test `Test that revoke cluster init bundle requires impacted clusters` doesn't show up in CI logs! 
Not sure why!